add2cal/add2cal.py

```diff
@@ -1,6 +1,6 @@
 from urllib import parse
```

Comment:
We should do the imports per our standards -- sorted, and with the four `ics` imports grouped together.

(We generally put `from` lines after `import` lines also, but that's not explicit in PEP8 that I can see.)